provision/provision-site.sh

```diff
@@ -73,7 +73,7 @@ function vvv_provision_site_nginx() {
   fi
   sed -i "s#{upstream}#${NGINX_UPSTREAM}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
 
-  if [[ $(is_utility_installed core tls-ca) ]]; then
+  if [[ $(/srv/config/homebin/is_utility_installed core tls-ca) ]]; then
```

Comment:
I would have rathered test this a bit further to get the command to run properly on the provisioners aswell without including the path.

Reply:
In the meantime is patched for who use the develop branch but I think that if we can do some tests an another pr can address this.